include/deal.II/matrix_free/cuda_matrix_free.h

```diff
@@ -153,26 +153,27 @@ namespace CUDAWrappers
 
     /**
      * Extracts the information needed to perform loops over cells. The
-     * DoFHandler and ConstraintMatrix describe the layout of degrees of
-     * freedom, the DoFHandler and the mapping describe the transformation from
-     * unit to real cell, and the finite element underlying the DoFHandler
-     * together with the quadrature formula describe the local operations.
+     * DoFHandler and AffineConstraints objects describe the layout of
+     * degrees of freedom, the DoFHandler and the mapping describe the
+     * transformation from unit to real cell, and the finite element
+     * underlying the DoFHandler together with the quadrature formula
+     * describe the local operations.
      */
     void
-    reinit(const Mapping<dim> &    mapping,
-           const DoFHandler<dim> & dof_handler,
-           const ConstraintMatrix &constraints,
-           const Quadrature<1> &   quad,
-           const AdditionalData    additional_data = AdditionalData());
+    reinit(const Mapping<dim> &             mapping,
+           const DoFHandler<dim> &          dof_handler,
+           const AffineConstraints<Number> &constraints,
```

Comment:
@Rombur Can you please test whether these changes on cuda supported matrix free work as expected? I have no ability to test my code changes.

Reply:
For me, all the CUDA tests pass after fixing the compile errors.